client/src/views/Simulation/components/ProvenanceGraph/ProvenanceGraphData.vue

```diff
@@ -1,32 +1,67 @@
 <template>
-  <section class="provenance-graph-data">
-    <h6>{{header}}</h6>
-  </section>
+  <div class="provenance-graph-data" ref="graph" />
```

Comment:
😢 we lost the `section` for a sad `div`.

Reply:
haha oops I changed it back to a section